### PR TITLE
Introduce pipenv for consistent environments

### PR DESCRIPTION
**Pull request by davehunt**
This patch introduces pipenv for consistent environments, and also updates the Docker image to use pipenv. This means that local development should be identical to deployment using the Docker images built by Circle CI. This also fixes an issue that's occuring in Circle CI where the server is responding with status code 500. Unfortunately this is currently considered acceptable as we only check that we get a response, and not the value of the status code.

### CODE REVIEW

**Comment by davehunt** [comment]
Closing/re-opening to hopefully trigger Circle CI build

**Comment by stephendonner** [comment]
Thanks, @davehunt - this should take care of https://github.com/mozilla/servicebook-web/issues/139

File: Makefile [hunk 1]
```diff
@@ -1,26 +1,12 @@
-HERE = $(shell pwd)
-VENV = .
-VIRTUALENV = virtualenv
-BIN = $(VENV)/bin
-PYTHON = $(BIN)/python
+.PHONY: build run
 
-INSTALL = $(BIN)/pip install --no-deps
+SERVICEBOOK ?= http://192.168.1.12:5000/api/
```
Comment (tarekziade) [inline Makefile]: where this ip comes from?

File: README.rst [hunk 2]
```diff
@@ -14,15 +14,32 @@ Service Book
...
 
+Hacking on Service Book
+-----------------------
+By default Service Book will use http://localhost:5000/api/ as the endpoint. If
+you want to use something else, you can set the `SERVICEBOOK` environment
+variable. For example, set this to http://servicebook.dev.mozaws.net/api/ to
+use the development environment. For convenience, you can store this locally in
+a `.env` file.
+
+To install the development environmet and start Service Book locally::
```
Comment (tarekziade) [inline README.rst]: typo. environment.

File: Makefile (hunk 1 above)
Comment (davehunt) [reply Makefile]: No idea. It's from https://github.com/mozilla/servicebook-web/blob/master/docker/Makefile#L3 😝

Comment (davehunt) [reply Makefile]: (by that I mean I didn't introduce it)

File: README.rst (hunk 2 above)
Comment (davehunt) [reply README.rst]: 👍 fixed

**Pull request merged**